.github/workflows/build.yml

```diff
@@ -12,11 +12,13 @@ concurrency:
 name: build
 jobs:
     build:
-        runs-on: ubuntu-latest
-        name: build
         env:
             # This is a fake connection string that is placed just for the compiler to stop whining
             POSTGRES_CONNECTION_STRING: 'postgresql://username:password@host/db_name'
+            PUBLIC_PREVIEW_HOST: ${{ secrets.PUBLIC_PREVIEW_HOST }}
+            PUBLIC_LIBRARY_ID: ${{ secrets.PUBLIC_LIBRARY_ID }}
```

Comment:
I don't think this particular variable exists. Is the purpose of having actual values (as opposed to dummy ones) here for running tests interacting with video in some sort of "test" library.

Reply:
The host exists, I forgot to add the library id, but thinking about it, there's no reason these couldn't be dummy values, I'm just in the habit of supplying real values.

Reply:
Totally fair. I think this approach may be useful if we end up having any tests relating to video services

Reply:
these have been replaced by dummy variables